README.md

```diff
@@ -126,4 +141,40 @@ qfusion/source$ make -j$(nproc)
 Executables are found in `qfusion/source/build` subdirectory.
 
 Having built the engine, you have run it from a directory which contains `basewsw` subdirectory with data files
-of Warsow 2.6+ (visit Warsow website/Warsow Discord for details).
\ No newline at end of file
+of Warsow 2.6+ (visit Warsow website/Warsow Discord for details).
+
+
+# Contributions
+
+We greatly appreciate any contributions to this project and want to make it as transparent as possible
+
+## Pull Requests
+We welcome your pull requests greatly.
+
+- Fork the repo and create your branch from main.
+- If you've added code that should be tested, add tests.
+- If you've changed APIs, update the documentation.
+- Ensure all tests and GitHub Actions workflows checks pass.
+- Make sure your code lints.
+
```

Comment:
This section is overly generic and does not apply to the current state of this repository (automation is currently non-functional)